sdcm/sct_config.py

```diff
@@ -1462,6 +1462,15 @@ class SCTConfiguration(dict):
         dict(name="stress_image", env="SCT_STRESS_IMAGE", type=dict_or_str,
              help="Dict of the images to use for the stress tools"),
 
+        dict(name="scylla_network_config", env="SCT_SCYLLA_NETWORK_CONFIG", type=list,
+             help="""Configure Scylla networking with single or multiple NIC/IP combinations.
+                  It must be defined for listen_address and rpc_address. For each address mandatory parameters are:
+                  - address: listen_address/rpc_address/broadcast_rpc_address/broadcast_address/test_communication
+                  - ip_type: ipv4 or ipv6"
```

Comment:
I think DNS isn't mentioned here

Reply:
it is not mandatory for now. It is not in use